src/components/Composer.vue

```diff
@@ -10,7 +10,7 @@
 					:value="selectedAlias"
 					:options="aliases"
 					label="name"
-					track-by="selectId"
+					:get-option-key="(option)=>option.selectId"
```

Comment:
we use track-by in other places too. is that a problem everywhere?

Reply:
apparently this is only needed if the objects haven an `id` and we don't want that value to be used for identification https://vue-select.org/api/props.html#getoptionkey

Reply:
`track-by` doesn't exist in [NcSelect](https://vue-select.org/api/props.html#getoptionkey). So If we have options we same Key, we would probably run into the same error

Reply:
By default key = option.id

Reply:
> apparently this is only needed if the objects haven an `id` and we don't want that value to be used for identification https://vue-select.org/api/props.html#getoptionkey

Yes in this case the alias has the same Id as the account